recipes/en_core_web_sm/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "en_core_web_sm" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
i'd probably recommend we have a more spacy specific prefix to this name here ?

Reply:
how about spacy-model-en_core_web_sm?